Gopkg.toml

```diff
@@ -1,5 +1,5 @@
 [[constraint]]
-  branch = "master"
+  branch = "dont-omit-zeroable-values"
```

Comment:
Change once other PR is merged

https://github.com/articulate/oktasdk-go/pull/28